src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mcfooter/MixedContentFooter.java

```diff
@@ -76,4 +109,48 @@ public int countArticleCards () {
     return articleLinks.size();
   }
 
+  public int countArticlesInExploreCard() {
+    wait.forElementVisible(exploreWikisCard);
+    return exploreWikisList.size();
+  }
+
+  public void clickOnViewAllLinkInDiscussions() {
+    wait.forElementVisible(viewAllLink).click();
+  }
+
+  public boolean isDiscussions() {
+    return discussionsBody.isDisplayed();
+  }
+
+  public String getUsername() {
```

Comment:
looking at this method when it is invoked it is hard to determine where the returned name comes from. Ideally this class should contain method `getDiscussionsCard()` which returns instance of separate class called `MCFdiscussionsCard` and this card could contain method `getUsername` (or even more descriptive name for this method would be `getFirstPostAuthor`)